README.rst

```diff
@@ -114,13 +114,12 @@ This virtual environment can be activated with:
 
 History
 -------
-This started as a stripped down version of `ouimeaux <https://github.com/iancmcc/ouimeaux>`_, but has since taken its own path.
+This started as a stripped down version of `ouimeaux <https://github.com/iancmcc/ouimeaux>`_, copyright Ian McCracken, but has since taken its own path.
```

Comment:
@iancmcc I'd like to make sure you're okay with how you're being credited here.
See also this thread: https://github.com/pavoni/pywemo/issues/244#issuecomment-792074486

Reply:
Absolutely fine. Thank you for checking!